src/update_handler.c

```diff
@@ -1175,6 +1178,10 @@ static gboolean img_to_boot_emmc_handler(RaucImage *image, RaucSlot *dest_slot,
 		goto out;
 	}
 
+	/* flush to block device before making eMMC boot partition read only */
+	fsync(out_fd);
```

Comment:
`fsync()` may fail, thus please also check the return value here and report an error if we fail